src/docs/framework-integration/angular.md

```diff
@@ -91,14 +91,14 @@ export const config: Config = {
   outputTargets: [
     angularOutputTarget({
       componentCorePackage: 'your-stencil-library-package-name',
-      directivesProxyFile: '../angular-workspace/projects/component-library/src/lib/stencil-generated/components.ts',
+      proxyDeclarationFile: '../angular-workspace/projects/component-library/src/lib/stencil-generated/components.ts',
       directivesArrayFile: '../angular-workspace/projects/component-library/src/lib/stencil-generated/index.ts',
     }),
   ],
 };
 ```
 
-> The `directivesProxyFile` is the relative path to the file that will be generated with all of the Angular component wrappers. You will replace the file path to match your project's structure and respective names. You can generate any file name instead of `components.ts`.
```

Comment:
I think it might be a good idea to call out the fact `directivesProxyFile` once existed, at least for the time being.  If someone were to upgrade their library, even though this is a breaking change, we don't have anything it terms of a migration guide to say "replace `directivesProxyFile` with `proxyDeclarationFile`

Reply:
Updated: https://stencil-site-c5lfk7ysm-ionic1.vercel.app/docs/angular#proxydeclarationfile-previously-directivesproxyfile

Let me know if there's a more preferred way of representing that. 

We are in pre 1.0, so it's a wild world anyways.